<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QPushButton" name="search_cat">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #b127be;
    border-radius: 10px;
    color: white;
    font-size: 16px; /* Tamaño del texto */
    padding: 8px; /* Espaciado interno */
}
QPushButton:pressed {
    background-color: #ffffff;
    color: black;
}
QPushButton:hover {
   border: 1px solid blue;
}
</string>
   </property>
   <property name="text">
    <string>Buscar categoría</string>
   </property>
  </widget>
  <widget class="QPushButton" name="myButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #b127be;
    border-radius: 10px;
    color: white;
    font-size: 16px; /* Tamaño del texto */
    padding: 8px; /* Espaciado interno */
}
QPushButton:pressed {
    background-color: #ffffff;
    color: black;
}
QPushButton:hover {
   border: 1px solid blue;
}
</string>
   </property>
   <property name="text">
    <string>Crear nueva categoría</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: Arial;
    font-size: 18px;
}</string>
   </property>
   <property name="text">
    <string>------------------------------------------</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3"/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>401</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/background.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #6527BE;
    color: white;
    font-size: 20px; /* Tamaño del texto */
    padding: 9px; /* Espaciado interno */
}</string>
   </property>
   <property name="text">
    <string>Selecciona una opción :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #b127be;
    border-radius: 10px;
    color: white;
    font-size: 16px; /* Tamaño del texto */
    padding: 8px; /* Espaciado interno */
}
QPushButton:pressed {
    background-color: #ffffff;
    color: black;
}
QPushButton:hover {
   border: 1px solid blue;
}
</string>
   </property>
   <property name="text">
    <string>En desarrollo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: Arial;
    font-size: 18px;
}</string>
   </property>
   <property name="text">
    <string>------------------------------------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: Arial;
    font-size: 18px;
}</string>
   </property>
   <property name="text">
    <string>------------------------------------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: Arial;
    font-size: 18px;
color: white
}</string>
   </property>
   <property name="text">
    <string>------------------------------------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: Arial;
    font-size: 18px;
color: white
}</string>
   </property>
   <property name="text">
    <string>------------------------------------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: Arial;
    font-size: 18px;
color: white
}</string>
   </property>
   <property name="text">
    <string>------------------------------------------</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>verticalLayoutWidget_3</zorder>
  <zorder>search_cat</zorder>
  <zorder>myButton</zorder>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
